cdiff.py

```diff
@@ -453,10 +534,7 @@ def _normalize(line):
         def _fit_with_marker(text, markup_fn, width, pad=False):
             """Wrap or pad input pure text, then markup"""
             if len(text) > width:
-                return markup_fn(text[:(width - 1)]) + wrap_char
-            elif pad:
-                pad_len = width - len(text)
-                return '%s%*s' % (markup_fn(text), pad_len, '')
+                return markup_fn(text)
```

Comment:
Expressions in two branches are identical now :).

Reply:
Ah right; I removed `_fit_with_marker` at all since it's not useful anymore.